utility/utils.py

```diff
@@ -716,47 +715,136 @@ def destroy_cluster(cluster_path):
         for volume in volumes:
             aws.detach_and_delete_volume(volume)
 
-        # Remove installer and tarball
-        os.remove(installer)
-        os.remove(tarball)
+        # Remove installer
+        delete_file(installer)
         return StatusOfTest.PASSED
 
     except Exception:
         log.error(traceback.format_exc())
         return StatusOfTest.FAILED
 
 
-def download_openshift_installer(version=defaults.INSTALLER_VERSION):
+def get_openshift_installer(version=defaults.INSTALLER_VERSION):
```

Comment:
@petr-balogh how can I rewrite this: `version=defaults.INSTALLER_VERSION` (and the other *defaults* variables like `CLIENT_VERSION` and `BIN_DIR`), to use the values from `default_config.yaml` file, instead of the old approach with `defaults` module?

Reply:
@dahorak you can define it in mentioned `default_config.yaml` in default section. But I think that this will be RUN specific, so we can directly maybe access this from RUN section and than it can be overwritten by passing custom config by --ocsci-conf custom-config.yaml by overwrite it in section it will be defined.

Reply:
@petr-balogh I've probably miss something, or there is some unexpected behaviour...

If the `DEPLOYMENT['installer_version']` construction used in `test_ocs_basic_install.py` file (in `test_deployment` function), it works as expected.
But if I try to import it and use similar construction in this file (`utility/utils.py`), it fails with `KeyError: 'bin_dir'`.

I've tried to log content of `DEPLOYMENT` on both places, and in the `test_....py` file it contains expected keys and values, but in `utils.py` file it is empty dictionary:
```
09:02:26 - MainThread - tests.ecosystem.deployment.test_ocs_basic_install - INFO - {'bin_dir': './bin/', 'client_version': '4.1.0-rc.3', 'installer_version': '4.1.0-rc.3'}
09:02:26 - MainThread - utility.utils - INFO - {}
```

Is there any required order of imported modules, or how the configuration should be initialized to be available also in other files, not only in the `test_...py` file?

Reply:
Ok, I've hopefully find solution for this. Instead of:
```
from ocsci.config import DEPLOYMENT
DEPLOYMENT['installer_version']
```
I have to use something like:
```
from ocsci import config
config.DEPLOYMENT['installer_version']
```

Reply:
I spent time on this issue yesterday and didn't come up yet with something with I am satisfied. So I am looking into this issue today as well and hope I will finally find some good solution for it.